Makefile

```diff
@@ -0,0 +1,51 @@
+all: setup build test lint
+
+.PHONY: setup
+setup:
+	stack setup
+	stack build --dependencies-only --test --no-run-tests
+	stack install --copy-compiler-tool hlint weeder
+
+.PHONY: build
+build:
+	stack build --fast --pedantic --test --no-run-tests
+
+.PHONY: test
+test:
+	stack build --fast --pedantic --test
+
+.PHONY: lint
+lint:
+	stack exec hlint
+	stack exec weeder
+
+.PHONY: clean
+clean:
+	stack clean
+
+.PHONY: check-nightly
+check-nightly: STACK_ARGUMENTS=--stack-yaml stack-nightly.yaml --resolver nightly
+check-nightly: setup build test
+
+NONEMPTY_ZIPPER_VERSION = \
+  v$(shell sed '/^version: \(.*\)$$/!d; s//\1/' ./package.yaml)
+
+## Release and tag bcp47 at the version in package.yaml
+.PHONY: release.nonempty-zipper
+release.bcp47: RELEASE_PACKAGE=nonempty-zipper
+release.bcp47: RELEASE_VERSION=$(NONEMPTY_ZIPPER_VERSION)
+release.bcp47: release
+
+RELEASE_PACKAGE ?=
+RELEASE_VERSION ?=
+
+## Release $(RELEASE_PACKAGE) at $(RELEASE_VERSION)
+.PHONY: release
+release:
```

Comment:
This is probably the only useful bit in the Makefile. I haven't decided the best way to share this across our libraries and the team though. Make targets in each specific repo is unlikely to scale well. Personally, I use [this](https://github.com/pbrisbin/dotfiles/blob/master/local/bin/stack-upload) for now, but I've been holding off on finding some way to share tooling like that across the team because the true Right Way is to have releases also automated as GitHub Actions.

Reply:
Cool, when it comes time to release I'll try that script.